pkg/restclient/client.go

```diff
@@ -168,24 +168,24 @@ func (r *Rest) Delete(url *types.URI) int {
 }
 
 // Get  http request
-func (r *Rest) Get(url *types.URI) (int, string) {
+func (r *Rest) Get(url *types.URI) (int, []byte, error) {
 	ctx := context.Background()
 	ctx, cancel := context.WithCancel(ctx)
 	defer cancel()
 	request, err := http.NewRequestWithContext(ctx, "GET", url.String(), nil)
 	if err != nil {
 		log.Errorf("error creating post request %v", err)
-		return http.StatusBadRequest, fmt.Sprintf("error creating post request %v", err)
+		return http.StatusBadRequest, nil, fmt.Errorf("error creating post request %v", err)
```

Comment:
All the error messages here talk about "post request", but actually this is a "get" request. The above function is "delete", but the comments are "post" again